Sezai/index.js

```diff
@@ -0,0 +1,23 @@
+// 20. datastructure array II
+
+const euroCities = ["Paris", "London", "Valletta", "Prague", "Rome"]
+console.log("euroCities ==> ", euroCities);
+
+euroCities[0]="Berlin"
+console.log("change 1st city to Berlin ==> ", euroCities);
+
+console.log("Length of euroCities", euroCities.length); // 5
+
+euroCities.pop()
+console.log("delete last item/city ==> ", euroCities);
+
+euroCities.push("Budapest")
+console.log("adding city to array ==> ", euroCities);
+
+euroCities.splice(1,2)
+console.log("euroCities ==> ", euroCities);
+
+const asianCities = ["Tokyo", "HongKong", "Okinawa"]
+console.log("asianCities ==> ", asianCities);
```

Comment:
Hey Sezai:) Please clone the array using array methods! 
https://www.freecodecamp.org/news/how-to-clone-an-array-in-javascript-1d3183468f6a/

Reply:
Or eiasier way is to clone it into another variable:)